pkg/rtc/participant.go

```diff
@@ -2230,6 +2231,9 @@ func (p *ParticipantImpl) addMediaTrack(signalCid string, sdpCid string, ti *liv
 	p.pendingTracks[signalCid].trackInfos = p.pendingTracks[signalCid].trackInfos[1:]
 	if len(p.pendingTracks[signalCid].trackInfos) == 0 {
 		delete(p.pendingTracks, signalCid)
+	} else {
+		p.pendingTracks[signalCid].queued = true
```

Comment:
@boks1971 it has been set to `queued` here if there are pending track info after the first one has been processed so it is ok to handle more than one pending tracks.

Reply:
Got it. Thank you @cnderrauber